redis/cache/redispipebp/monitored.go

```diff
@@ -123,6 +148,7 @@ func (s MonitoredSync) Send(ctx context.Context, r redis.Request) (result interf
 		redisprom.DeploymentLabel: "", // We don't have that info
 	})
 	active.Inc()
+	s.UpdatePeakConnection(active)
```

Comment:
I don't think this is really right, or at least it's not measuring connections. Redispipe uses a single connection per node and batches requests into a pipeline so when you send a new request, you aren't using another connection.

Reply:
+1 

It could also be more than one connection depending on how the client is configured :thinking:

Reply:
removed.